docs/index.rst

```diff
@@ -18,6 +18,7 @@ Contents:
    list_of_services
    common_tasks
    rest_api
+   submitting_pulse_data
```

Comment:
Yep, good call.  I reorganized things a bit and added a brief synopsis to the ref links for pulse vs. API.